packages/dds/tree/src/test/feature-libraries/flex-map-tree/mapTreeNode.spec.ts

```diff
@@ -0,0 +1,222 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+
+import {
+	FieldKinds,
+	FlexFieldSchema,
+	SchemaBuilderBase,
+	getOrCreateMapTreeNode,
+} from "../../../feature-libraries/index.js";
+import { TreeStatus } from "../../../../dist/index.js";
+import { EmptyKey, FieldKey, MapTree } from "../../../core/index.js";
+import { leaf as leafDomain } from "../../../domains/index.js";
+import { brand } from "../../../util/index.js";
+
+describe("MapTreeNodes", () => {
```

Comment:
Nit: do we usually pluralize the suite names like this? I would expect `MapTreeNode` personally, since the suite is testing a singular concept.

Reply:
I think I've tended to do this in the past. I've never thought much about it. I always try to make the test cases be a readable sentence from highest `describe` to lowest `it`. So you end up with chains like "MapTreeNodes get their fields" But we could also do "MapTreeNode gets its fields". Just sounds a little funny, since it makes MapTreeNode into a proper noun. "Hey, that's my buddy MapTreeNode over there..." :)

I'm fine either way - I'll plan on keeping it as is, but if you prefer the other way just reply below and I'll change it.